src/managers/contact_manager.py

```diff
@@ -134,3 +134,24 @@ def search_by_phone_number(self, phone_number: str) -> List[Contact]:
                     matching_contacts.append(contact)
 
         return matching_contacts
+    
+    def handle_command(self, command: str) -> None:
```

Comment:
@AndreyShvachko hello!
It is necessary to transfer the handler logic to the command_handlers.py file. You can also remove the handle_command function, the logic in the middle of this function needs to be implemented at the interface level in main.py

Reply:
Before making changes, you can pull the fresh code from the main branch

Reply:
@MHryhoriev Hello!
So, should I create a new file? Not clearly understand in which folder should I put it. 

Thak you!

Reply:
This file is already created in the utils folder, you just need to update the code from the main branch and everything will be there.